app/models/application_record.rb

```diff
@@ -0,0 +1,32 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
+
+  include JwtToken
+
+#    before_action :authenticate_user
+
+    # before_action :set_locale
+    private
+        def authenticate_user
```

Comment:
this is a violation of the MVC pattern rules, this is a controller based function  why is it included in application reocrd? same  goes for the rest of the file